proto/minder/v1/minder.proto

```diff
@@ -2290,33 +2332,78 @@ message Profile {
         // id is optional and use for updates to match upserts as well as read operations. It is ignored for creates.
         string id = 1;
         // entity is the entity to select.
-        string entity = 2;
+        string entity = 2 [
+            (buf.validate.field).string = {
+                pattern: "^[a-z]+(_[a-z]+)*$",
+                min_len: 1,
+                max_len: 200,
+            },
+            (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+        ];
         // expr is the expression to select the entity.
-        string selector = 4;
+        string selector = 4 [
+            (buf.validate.field).string = {
+                pattern: "^[a-zA-Z_][a-zA-Z0-9_]*(?:\\.[a-zA-Z_][a-zA-Z0-9_]*)*(?:\\[\\d+\\])?$",
```

Comment:
I don't understand this.  This is a CEL expression, correct?

It feels like this might be a special case where we say "this will be parsed as a CEL expression", because I'm nervous about encoding the allowed CEL characters in a regex.  (In particular, I suspect they allow for greater-than and less-than expressions...)

Reply:
Makes sense 👍 So we can leave it without validation for now I suppose?